torch/csrc/jit/script/module.h

```diff
@@ -184,6 +184,23 @@ struct TORCH_API Module {
     return module_object()->setAttr(name, v);
   }
 
+  void setattr(const std::string& name, IValue v) {
```

Comment:
The addition of this method is suspicious to me. What's the difference between `set_attribute` and `setattr`? It appears the only difference is that there is a subtyping check, which should exist on `set_attribute` as well (the current implementation of `set_attribute` is incorrect.)